pkg/server/delta/v3/watches.go

```diff
@@ -0,0 +1,50 @@
+package delta
+
+import (
+	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
+	"github.com/envoyproxy/go-control-plane/pkg/cache/v3"
+	"github.com/envoyproxy/go-control-plane/pkg/server/stream/v3"
+)
+
+// watches for all delta xDS resource types
+type watches struct {
+	deltaResponses    map[string]watch
+	deltaTerminations map[string]chan struct{}
+
+	// Organize stream state by resource type
+	deltaStreamStates map[string]stream.StreamState
+
+	// Opaque resources share a muxed channel. Nonces and watch cancellations are indexed by type URL.
```

Comment:
This says we index by type URL but this comment is just on a chan, so not clear where this indexing is happening? Maybe clarify?

Reply:
Ah oops, actually that comment is old, i'll remove it since it no longer applies.

Reply:
That previously came from here:
```go
	// Opaque resources share a muxed channel. Nonces and watch cancellations are indexed by type URL.
	responses     chan cache.Response
	cancellations map[string]func()
	nonces        map[string]string
	terminations  map[string]chan struct{}
```